### PR TITLE
case-lib: improve log readability

### PR DESCRIPTION
**Pull request by aiChaoSONG**
check the log for lib.sh, hijack.sh and pipeline.sh, refine some of the log to eliminate ambiguity and unclear expression.

### CODE REVIEW

File: case-lib/pipeline.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ func_pipeline_export()
 {
```
Comment (paulstelian97) [inline case-lib/pipeline.sh]: Typo in commit message (and PR title), should be "improve".

File: case-lib/pipeline.sh [hunk 2]
```diff
@@ -21,9 +21,9 @@ func_pipeline_export()
         elif [ -f "$f" ]; then
             f=$(realpath $f)    # relative path -> absolute path
         else
-            dlogw "Couldn't find target TPLG file $f needed to run $0" && exit 1
+            dlogw "Topology $f is not found, unable to run command: $0" && exit 1
         fi
-        dlogi "${BASH_SOURCE[-1]} using $f as target TPLG to run the test case"
+        dlogi "${BASH_SOURCE[-1]} will use TPLG:$f to run the test case"
```
Comment (wenqingfu) [inline case-lib/pipeline.sh]: "TPLG:"->topology to align with other changes

File: case-lib/pipeline.sh [hunk 3]
```diff
@@ -45,22 +45,22 @@ func_pipeline_export()
         opt=$opt" -b"
         for key in ${!TPLG_IGNORE_LST[@]}
         do
-            dlogi "Catch ignore option from TPLG_IGNORE_LST will ignore '$key=${TPLG_IGNORE_LST[$key]}' for $tplg_str"
+            dlogi "Ignore list for pipeline found, will ignore '$key=${TPLG_IGNORE_LST[$key]}' in test case"
```
Comment (wenqingfu) [inline case-lib/pipeline.sh]: "Ignore list for pipeline found" -> "pipeline list to ignore is specified,"

File: case-lib/lib.sh [hunk 4]
```diff
@@ -107,11 +107,11 @@ func_lib_restore_pulseaudio()
         wait_t=$[ $wait_t + 1 ]
         sleep 1s
         if [ $wait_t -ge $timeout ]; then
-            dlogw "Restore pulseaudio wait too long: $timeout"
+            dlogw "Pulseaudio not restored in $timeout seconds"
```
Comment (wenqingfu) [inline case-lib/lib.sh]:
```suggestion
            dlogw "Time out. Pulseaudio not restored in $timeout seconds"
```

File: case-lib/hijack.sh [hunk 5]
```diff
@@ -37,7 +37,7 @@ function exit()
     IFS="$OLD_IFS"
     # now force kill target process which maybe block the script quit
     if [ ${#cmd_lst[@]} -gt 0 ]; then
-        dlogw "${BASH_SOURCE[-1]} load exit still have those process exist:"
+        dlogw "Process(es) started by ${BASH_SOURCE[-1]} not fully exited/killed, kill these process:"
```
Comment (wenqingfu) [inline case-lib/hijack.sh]:
```suggestion
        dlogw "Process(es) started by ${BASH_SOURCE[-1]} are still active, kill these process(es):"
```

File: case-lib/pipeline.sh (hunk 1 above)
Comment (aiChaoSONG) [reply case-lib/pipeline.sh]: Oops, Hand is faster than the brain! Thanks for the catch

File: case-lib/lib.sh (hunk 4 above)
Comment (aiChaoSONG) [reply case-lib/lib.sh]: all done

**Comment by xiulipan** [comment]
@aiChaoSONG The patch looks good, thanks for clean up the logs.
It seems we have some conflict here. Please rebase and we can merge this part fix.

**Comment by aiChaoSONG** [comment]
@xiulipanYes, we got conflict after kelly's clean up merged, and now, we got even more because of #120, I want to wait #120 merged, and do the rebase, or I have to do rebase for this and #120.

**Comment by aiChaoSONG** [comment]
@xiulipan Can we get this merged? day by day, any change to lib.sh, pipeline.sh, hijack.sh will get this PR conflict to merge. since this is only log improvement and Wubin, Michael and you have reviewed, let's merge this ASAP

**Pull request merged**